trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/TrinoAwsProxyServerModule.java

```diff
@@ -121,6 +122,9 @@ protected void setup(Binder binder)
         });
         newSetBinder(binder, com.fasterxml.jackson.databind.Module.class).addBinding().toProvider(JsonIdentityProvider.class).in(Scopes.SINGLETON);
 
+        // RequestRewriter binder
+        newOptionalBinder(binder, S3RequestRewriter.class);
```

Comment:
Can we bind a default NOOP implementation (with a provider that also logs)? This way `TrinoS3ProxyClient` can request a `S3RequestRewriter` instead of its optional